### PR TITLE
Site admin / repos - uniform repo icon size.

### PR DESCRIPTION
**Pull request by cbart**
Part of https://github.com/sourcegraph/sourcegraph/issues/39327

When working on https://github.com/sourcegraph/sourcegraph/issues/39327, I noticed that repo icons were resizing in a diversified manner:

Screenshot 2023-03-29 at 3 17 23 PM

This change introduces a `min-width` and `min-height` to the icon, so that it does not allow to be shrinked:

Screenshot 2023-03-29 at 3 17 41 PM

## Test plan

Visual assessment.

## App preview:

- [Web](https://sg-web-cb-site-admin-repos-uniform-icons.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Review by varsanojidan — APPROVED** [review]
Guest appearance by @indradhanush

**Review by st0nebraker — APPROVED** [review]

**Comment by cbart** [comment]
> Guest appearance by @indradhanush

I did not think that @indradhanush is such an #emacsGuy

Screenshot 2023-03-29 at 7 48 57 PM

**Comment by cbart** [comment]
> > Guest appearance by @indradhanush
> 
> I did not think that @indradhanush is such an #emacsGuy
> 
> Screenshot 2023-03-29 at 7 48 57 PM

That's pretty cool.

**Comment by cbart** [comment]
@cody just fix link for me pls

File: client/web/src/site-admin/RepositoryNode.module.scss [hunk 1]
```diff
@@ -44,3 +44,8 @@
     max-height: 300px;
     overflow-y: auto;
 }
+
+.repo-icon {
+    min-height: 24px;
```
Comment (sashaostrikov) [inline client/web/src/site-admin/RepositoryNode.module.scss:49]: Nit: To me it looks a bit big on a screenshot (at least bigger than before), maybe we can fix the smallest size?

**Review by sashaostrikov — APPROVED** [review]
LGTM! Left one comment :)

**Comment by mrnugget** [comment]
> were resizing in a diversified manner

that's one way to say "looks broken" 😄

File: client/web/src/site-admin/RepositoryNode.module.scss (hunk 1 above)
Comment (cbart) [reply client/web/src/site-admin/RepositoryNode.module.scss]: :brat: you're right. Let's do 16.
Screenshot 2023-03-30 at 2 25 43 PM

**Pull request merged**